src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -536,6 +536,10 @@ def getJobParameters(self, jobList):
             ad['My.CMS_JobType'] = classad.quote(job['task_type'])
             ad['My.CMS_Type'] = classad.quote(activityToType(job['activity']))
             ad['My.CMS_RequestType'] = classad.quote(job['requestType'])
+            if job.get('campaignName'):
```

Comment:
Should we cover the case where campaign has been mistakenly defined as an empty string (instead of propagating it as undefined)? Example, I would suggest:
```
if job.get('campaignName') is None:
   ad['My.CMS_CampaignName'] = undefined
else:
   ad['My.CMS_CampaignName'] = classad.quote(job['campaignName'])
```